WordPress/src/androidTest/java/org/wordpress/android/e2e/StatsNewTests.kt

```diff
@@ -0,0 +1,46 @@
+package org.wordpress.android.e2e
+
+import androidx.test.espresso.Espresso
+import androidx.test.espresso.matcher.ViewMatchers
+import dagger.hilt.android.testing.HiltAndroidTest
+import org.junit.After
+import org.junit.Assume
+import org.junit.Before
+import org.junit.Test
+import org.wordpress.android.BuildConfig
+import org.wordpress.android.R
+import org.wordpress.android.e2e.pages.MySitesPage
+import org.wordpress.android.support.BaseTest
+import org.wordpress.android.support.ComposeEspressoLink
+import org.wordpress.android.support.WPSupportUtils
+import org.wordpress.android.util.StatsKeyValueData
+import org.wordpress.android.util.StatsMocksReader
+import org.wordpress.android.util.StatsVisitsData
+
+@HiltAndroidTest
+class StatsNewTests : BaseTest("new-stats-feature-response.json") {
```

Comment:
We named this class as `StatsNewTests` here. and the function in  `StatsPage` as  `hasNewStatTabs()`. This "new" screen will be default soon. 😁
What if change the name of this class `StatsTest` and rename the old screen `StatsGranularTabsTest`? Wdyt?

Reply:
Haha. It's just naming. But I don't mind changing it. Will do it.